contrib/build-static.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash -xeu
+
+DIR="${DIR:=$(realpath `dirname "${0}"`)}"
+
+REPO_MUSL="https://git.launchpad.net/musl"
+REPO_LIBTIRPC="https://salsa.debian.org/debian/libtirpc.git"
+REPO_LIBNSL="https://github.com/thkukuk/libnsl.git"
+REPO_LIBUV="https://github.com/libuv/libuv.git"
+REPO_LIBLZ4="https://github.com/lz4/lz4.git"
+REPO_SQLITE="https://github.com/sqlite/sqlite.git"
+REPO_DQLITE="https://github.com/cole-miller/dqlite.git"
+
+TAG_MUSL="v1.2.4"
+TAG_LIBTIRPC="upstream/1.3.3"
+TAG_LIBNSL="v2.0.1"
+TAG_LIBUV="v1.48.0"
+TAG_LIBLZ4="v1.9.4"
+TAG_SQLITE="version-3.45.1"
+TAG_DQLITE="musl-ci"
```

Comment:
Do we need to update the tags manually when new versions come out? Does it make more sense to use the latest version instead?

Reply:
Hmm, it probably makes sense to keep in sync with Juju and MicroK8s on the tested version of musl, but latest versions of everything else sounds good to me. We could merge this with the `latest-deps` CI job.

Reply:
Updated to use latest versions of dependencies except musl, which is pinned at v1.2.4.